core/src/main/java/io/confluent/rest/handlers/SniUtils.java

```diff
@@ -28,9 +28,10 @@
 
 public class SniUtils {
   public static String getSniServerName(Request baseRequest) {
-    EndPoint endpoint = baseRequest.getHttpChannel().getEndPoint();
```

Comment:
@bob-barrett can you sanity check this logic? This might impact the v4 everywhere logic. 
Context: rest-utils is being moved to jetty 12 by @apurushotama, bunch of APIs are changing from jetty 9 to 12 versions.

PS - Looks alright at high level.

Reply:
@bob-barrett reviewed separately, closing this.